package.json

```diff
@@ -45,7 +46,7 @@
 		"@libsql/client": "^0.15.9",
 		"@types/node": "^24.0.4",
 		"better-auth": "^1.2.10",
-		"remult": "^3.0.5",
+		"remult": "^3.0.6-next.0",
```

Comment:
I can do this when 3.0.6 is ready .. unless you think there may be a breaking change here?